x/ibc/04-channel/client/cli/tx.go

```diff
@@ -421,13 +428,14 @@ $ %s tx ibc channel handshake [client-id] [port-id] [chan-id] [conn-id] [cp-clie
 
 			viper.Set(flags.FlagChainID, cid1)
 			msgUpdateClient = ibcclient.NewMsgUpdateClient(clientid1, header, ctx1.GetFromAddress())
-			fmt.Printf("%v <- %-23v", cid1, msgUpdateClient.Type())
+			if err := msgUpdateClient.ValidateBasic(); err != nil {
+				return err
+			}
+
 			res, err = utils.CompleteAndBroadcastTx(txBldr1, ctx1, []sdk.Msg{msgUpdateClient}, passphrase1)
 			if err != nil || !res.IsOK() {
-				fmt.Println(res)
 				return err
 			}
-			fmt.Printf(" [OK] txid(%v) client(%v)\n", res.TxHash, clientid1)
```

Comment:
Again, here I would like to leave this printing in the handshake to allow users to understand which transactions are being run against which chain during the handshake

Reply:
that is why the commands should be separate imo. This should be resolved with proper documentation